l10n_br_pos/__manifest__.py

```diff
@@ -0,0 +1,67 @@
+# © 2016 KMEE INFORMATICA LTDA (https://kmee.com.br)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Ponto de venda adaptado a legislação Brasileira",
+    "version": "14.0.1.0.0",
+    "author": "KMEE, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/l10n-brazil",
+    "license": "AGPL-3",
+    "category": "Point Of Sale",
+    "development_status": "Alpha",
+    "maintainers": ["mileo", "sadamo", "gabrielcardoso21", "lfdivino"],
```

Comment:
sera se @sadamo e gabrielcardoso21 serão mantenedores ativos mesmo?
Eles não estão mexendo mais nada no projeto... 

- @sadamo tem 0 contribuição github em 2023, 10 em 2022, 0 em 2021... https://github.com/sadamo
- @gabrielcardoso21 0 contribuição no projeto em 2023 e 0 em 2022... https://github.com/gabrielcardoso21

Eu acho ruim passar a imagem que tem mantenedor fantasma quando não tem e que no mesmo tempo tem pessoas fazendo trabalho e que não tem a visibilidade. Ja tem vários módulos no repo que pessoas da KMEE estão cadastradas como mantenedor e faz uns 2 anos que não revisaram nem um PR (exemplo: l10n_br_nse)...

cc @renatonlima @gabrielcardoso21 @mbcosta @antoniospneto @felipemotter

Reply:
O @sadamo esta liderando projetos com @shopinvader e esta um pouco afastado mesmo. 

Mas como foi ele quem criou o módulo na v8 vale a pena tirar? Existe alguma regra pra isso?

Reply:
Acho que a pergunta é.. Quais pessoas de fato serão mantenedores do módulo. Se o @sadamo vai fazer este papel não vejo porque tirar, mas se não for ai melhor definir outra pessoa independete da autoria. Acho que o fato de ser o autor necessáriamente não obriga que seja aquela pessoa que vai manter (n casos assim).

Reply:
Beleza, por hora que esta mantendo já esta manifesto. Inclusive fizemos ajustes.